packages/palette/src/elements/Modal/Modal.story.tsx

```diff
@@ -4,43 +4,63 @@ import { Button } from "../Button"
 import { Modal } from "./Modal"
 
 storiesOf("Components/Modal", module)
-  .add("Default", () => {
-    return (
-      <Modal show onClose={() => null}>
-        Some example content
-      </Modal>
-    )
-  })
-  .add("With title", () => {
-    return (
-      <Modal show title="Modal Title" onClose={() => null}>
-        Some example content
-      </Modal>
-    )
-  })
-  .add("With logo", () => {
-    return (
-      <Modal show title="Modal Title" hasLogo onClose={() => null}>
-        Some example content
-      </Modal>
-    )
-  })
-  .add("Modal with fixed button", () => {
-    return (
-      <Modal
-        show
-        title="Modal Title"
-        onClose={() => null}
-        FixedButton={<Button width="100%">Click me</Button>}
-      >
-        Some example content
-      </Modal>
-    )
-  })
-  .add("Wide modal", () => {
-    return (
-      <Modal show title="Modal Title" isWide onClose={() => null}>
-        Some example content
-      </Modal>
-    )
-  })
+  .add(
+    "Default",
+    () => {
+      return (
+        <Modal show onClose={() => null}>
+          Some example content
+        </Modal>
+      )
+    },
+    { chromatic: { delay: 500 } }
+  )
+  .add(
+    "With title",
+    () => {
+      return (
+        <Modal show title="Modal Title" onClose={() => null}>
+          Some example content
+        </Modal>
+      )
+    },
+    { chromatic: { delay: 500 } }
+  )
+  .add(
+    "With logo",
+    () => {
+      return (
+        <Modal show title="Modal Title" hasLogo onClose={() => null}>
+          Some example content
+        </Modal>
+      )
+    },
+    { chromatic: { delay: 500 } }
+  )
+  .add(
+    "Modal with fixed button",
+    () => {
+      return (
+        <Modal
+          show
+          title="Modal Title"
+          onClose={() => null}
+          FixedButton={<Button width="100%">Click me</Button>}
+        >
+          Some example content
+        </Modal>
+      )
+    },
+    { chromatic: { delay: 500 } }
+  )
+  .add(
+    "Wide modal",
+    () => {
+      return (
+        <Modal show title="Modal Title" isWide onClose={() => null}>
+          Some example content
+        </Modal>
+      )
+    },
+    { chromatic: { delay: 500 } }
+  )
```

Comment:
OK well these look good now and just have to be accepted this one time

![](http://static.damonzucconi.com/_capture/Ev6JMWB4v533.png)